docs/publish/deploy-office-add-in-sso-to-azure.md

```diff
@@ -273,6 +273,8 @@ Sideload the **./dist/manifest.xml** file and test the functionality of the add-
 
 If you encounter any deployment issues, see the [Azure App Service troubleshooting documentation](/troubleshoot/azure/app-service/welcome-app-service). If you use central deployment, or plan to deploy to App Source, we recommend you [validate the Office Add-in's manifest](../testing/troubleshoot-manifest.md#office-store-validation) using the '-p' option for production.
 
+[!INCLUDE [deploy-updates-that-require-admin-consent](../includes/deploy-updates-that-require-admin-consent.md)]
```

Comment:
Does the header the include uses work here? Everything else in that article is an instruction, whereas this is more of a next step. Consider adding this under the "Build and deploy" section as an H3-level thing.

Reply:
thanks! Moved.